docs/reference/method/MongoDBCollection-watch.txt

```diff
@@ -0,0 +1,50 @@
+================================
+MongoDB\\Collection::watch()
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Collection::watch()
+
+   Executes a :manual:`change stream </changeStreams>` operation on the
+   collection.
+
+   .. code-block:: php
+
+      function watch(array $pipeline = [], array $options = []): ChangeStream
```

Comment:
The return type here should be a FQCN (fully qualified class name).

It's not required for PHP doc blocks, since those can capitalize on the `use` statements in the same file.